### PR TITLE
Leaf/b1.2_http_post_call_handling_new

### PR DESCRIPTION
**Pull request by bluesimp1102**
Most of the changes are pretty much the same as the last PR, I added some new stuff so the account module can be added correctly to the new main API module with the refactored structures.

Issue https://github.com/LetsMesh/Site/issues/67

### CODE REVIEW

**Comment by misslame** [comment]
Looks like you have merge conflicts. Could you also link the issue in the PR.

**Comment by misslame** [comment]
You also need to target `dev` branch not main

**Comment by bluesimp1102** [comment]
I changed the target to dev branch and linked the issue to the original PR post. 

Issue https://github.com/LetsMesh/Site/issues/67

**Comment by misslame** [comment]
I'll review when merge conflicts are fixed

File: mesh/accounts/models.py [hunk 1]
```diff
@@ -1,14 +1,24 @@
+# in accounts folder: (accounts.models) models.py
 from django.db import models
 
-class Account(models.Model):
+class Accounts(models.Model):
```
Comment (misslame) [inline mesh/accounts/models.py:4]: We recently found out that django admin adds an S to the table name already so the S becomes redundant.

File: mesh/accounts/views.py [hunk 2]
```diff
@@ -1 +1,105 @@
...
+from django.core import serializers
+from django.utils.decorators import method_decorator
+from django.views import View
+
+from .models import Accounts
+from ..utils.validate_data import validate_json_and_required_fields
+import json
+
+
+@method_decorator(csrf_exempt, name='dispatch')
```
Comment (misslame) [inline mesh/accounts/views.py:12]: Why is it csrf exempt?

Comment (misslame) [reply mesh/accounts/views.py:12]: Same with the following

File: mesh/exceptions/InvalidJsonFormat.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+from .BaseCustomException import BaseCustomException
+
+class InvalidJsonFormat(BaseCustomException):
```
Comment (misslame) [inline mesh/exceptions/InvalidJsonFormat.py:3]: I love the exception handling organization with the base and child specific class

**Review by misslame — APPROVED** [review]

**Comment by misslame** [comment]
Overall great work. Some comments but minor.

**Comment by misslame** [comment]
I apologize but please retarget to `dev_new`

File: mesh/accounts/views.py (hunk 2 above)
Comment (bluesimp1102) [reply mesh/accounts/views.py:12]: from what I learned about it, CSRF from Django helps you protect the website/application or maybe the API from receiving malicious requests or spam requests.

**Pull request closed without merging**